model/src/main/kotlin/Issue.kt

```diff
@@ -53,7 +54,13 @@ data class Issue(
     /**
      * The issue's severity.
      */
-    val severity: Severity = Severity.ERROR
+    val severity: Severity = Severity.ERROR,
+
+    /**
+     * The affected file or directory the issue is limited to, if any.
+     */
+    @JsonInclude(JsonInclude.Include.NON_NULL)
+    val affectedPath: String? = null
```

Comment:
I'm not so happy with the name. `affectedPath` sounds like the file at this location is affected by the issue. For example, if a file system is corrupt, then files can be affected by this. But with the example of a ScanCode timeout, the timeout does not affect the file at all, the file is merely the cause of the issue because ScanCode cannot handle it for some reason.
Since we do not know what kinds of issues might make use of this property in future, shall we just call it `filePath` or `path`?

Reply:
> But with the example of a ScanCode timeout, the timeout does not affect the file at all

Well, I'm reading it differently, like "What thing was affected by the timeout? The file, because it's contents were not properly scanned."

> Since we do not know what kinds of issues might make use of this property in future, shall we just call it `filePath` or `path`?

As mentioned on Slack I was against `filePath` as also directories could be affected, and `path` sounds overly generic, thought I could live with it. Maybe `relatedPath`?

Reply:
I remove myself / my preference from this discussion and will implement what you guys agree @mnonnenmacher  @sschuberth.

Reply:
If both of you are happy with the current name please just go ahead.